### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by brookewursten**
Added mappedbatches clients endpoints, tests and various fixes to models and DTOs.  Most recent working version of the admin branch

### CODE REVIEW

**Comment by kMaths** [comment]
Please write a test for findByAdminId. I think just that should bring the code coverage up above 70%. Also, please include tests for all parts of try-catch blocks and if-else blocks.

File: src/main/resources/application_local.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+server.port=9011
+
+#These don't work, currently figuring out a solution to use environmental variables
+#spring.datasource.url=${System.getenv("url")}
+#spring.datasource.username=${System.getenv("username")}
+#spring.datasource.password=${System.getenv("password")}
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```
Comment (kMaths) [inline src/main/resources/application_local.properties:9]: We don't want the database to be this

File: src/test/java/com/engagement/ClientEngagementPortalBackApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.engagement;
```
Comment (kMaths) [inline src/test/java/com/engagement/ClientEngagementPortalBackApplicationTests.java:1]: This can be deleted

**Comment by kMaths** [comment]
Let's get this up to 70% before we merge

**Review by TyroneV — APPROVED** [review]

**Pull request merged**